tests/sentry/mediators/sentry_apps/test_updater.py

```diff
@@ -45,3 +45,18 @@ def test_updates_webhook_url(self):
         self.updater.webhook_url = 'http://example.com/hooks'
         self.updater.call()
         assert self.sentry_app.webhook_url == 'http://example.com/hooks'
+
+    def test_updates_redirect_url(self):
```

Comment:
Is it useful to have an endpoint test that exercises the serializer?

Reply:
I think each endpoint suite should have one test that ensures the entire payload looks as we expect.